<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>669</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>844</width>
    <height>634</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>844</width>
    <height>669</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#signup{
	border-image: url(:/images/Myresources/background.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>217</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>92</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{
	border: 2px;
	border-radius: 10px;
	background-color: rgba(30, 30, 30, 150);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1" rowspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Berlin Sans FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>SignUp</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>First Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Last Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>10</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>date of birth</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="styleSheet">
               <string notr="true">#lineEdit{
color :rgb(255, 255, 255);
background : transparent;
border: none;
border-bottom: 1px solid ;

}</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="styleSheet">
               <string notr="true">#lineEdit_2 {
background : transparent;
border: none;
border-bottom: 1px solid ;
	color: rgb(255, 255, 255);
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="styleSheet">
               <string notr="true">#lineEdit_3{
background : transparent;
border: none;
border-bottom: 1px solid ;
	color: rgb(255, 255, 255);
}
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_4">
              <property name="styleSheet">
               <string notr="true">#lineEdit_4{
	background : transparent;
	border: none;
	border-bottom: 1px solid ;
	color: rgb(255, 255, 255)
}</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignHCenter">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#label_6{

background : transparent;
border: none;
color:rgb(255, 255, 255)
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCalendarWidget" name="calendarWidget">
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>163</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#calendarWidget {
    background-color: rgba(50, 50, 50, 180); /* translucent grey */
    border: 1px solid rgba(0, 0, 0, 150);
    border-radius: 5px;
}

#calendarWidget QToolButton { /* month/year navigation buttons */
    background-color: rgba(80, 80, 80, 180);
    color: white;
}

#calendarWidget QSpinBox { /* year spinbox */
    background-color: rgba(80, 80, 80, 180);
    color: white;
}

#calendarWidget QAbstractItemView { /* day cells */
    background-color: rgba(50, 50, 50, 180);
    color: white;
    selection-background-color: rgba(100, 100, 100, 200);
    selection-color: white;
}

#calendarWidget QAbstractItemView::item:hover { /* hover effect on days */
    background-color: rgba(120, 120, 120, 200);
}
</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#pushButton
{
color: rgb(0, 255, 255);
background-color: transparent;
border: none;
}</string>
              </property>
              <property name="text">
               <string>back</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="font">
               <font>
                <family>Berlin Sans FB</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">#pushButton_2{

color: rgb(0, 255, 255);
background-color: transparent;
border: none;

}</string>
              </property>
              <property name="text">
               <string>save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>331</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" rowspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>331</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>218</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
